src/bot/cogs/logging.py

```
@@ -0,0 +1,159 @@
+                "event_name": event_name,
+                "log_channel_id": channel_id_str,
+                "webhook_url": webhook_url,
+                "ignored_channels": event_config.ignored_channels
+            }
+            await self.bot.api.set_logging_event(str(guild.id), event_name, save_data)
+            return webhook_url
+        except Exception as e:
+            logging.error(f"Webhook Error: {e}")
+            return None
+
+    async def _process_log(self, guild: discord.Guild, event_name: str, embed: discord.Embed, trigger_channel_id: str = None):
```

Ruff (0.15.7)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)